client/packages/inventory/src/Stocktake/api/hooks/line/useStocktakeRows.ts

```diff
@@ -15,9 +15,12 @@ const getStocktakeItems = (lines: StocktakeLineFragment[]) =>
     } as StocktakeSummaryItem;
   });
 
-export const useStocktakeRows = () => {
+export const useStocktakeRows = (itemCode?: string) => {
   const { data: stocktake } = useStocktake.document.get();
-  const { data: lineData, isLoading } = useStocktakeLines(stocktake?.id ?? '');
+  const { data: lineData, isLoading } = useStocktakeLines(
```

Comment:
Maybe we need something like this?
```suggestion
  const { data: lineData, isLoading } = useStocktakeLinesForItem(
```